### PR TITLE
Sync Static's env vars with Puppet in integration.

### PR DESCRIPTION
**Pull request by sengi**
- Add Google Tag Manager ID.    
- Add values for Google Tag Manager preview functionality.    
- Add bearer token for Static to talk to Publishing API. This is only used in a Rake task.
    
I double-checked with UX Measurement team about the GA/GTM values and they confirmed that they're definitely not supposed to be secrets.

Tested:

- `helm install chris-test-static ../generic-govuk-app --values <(helm template . --values values-integration.yaml |yq e '. |select(.metadata.name == "static").spec.source.helm.values') --set sentry.createSecret=false` (sentry.createSecret=false is just to avoid a name conflict).
- `k get deploy chris-test-static`
- `k logs deploy/chris-test-static`
- `helm uninstall chris-test-static`.

### CODE REVIEW

File: charts/app-config/templates/external-secrets/static/google-tag-manager-preview.yaml
Comment (nsabri1) [inline charts/app-config/templates/external-secrets/static/google-tag-manager-preview.yaml:11]: Is it the case that this secret will no exist in Production?

Comment (sengi) [reply charts/app-config/templates/external-secrets/static/google-tag-manager-preview.yaml:11]: That's right — doesn't exist in current Production and won't need to in ours either.

Comment (sengi) [reply charts/app-config/templates/external-secrets/static/google-tag-manager-preview.yaml:11]: It's for testing changes to Google Tag Manager configuration without applying them to the live website, hence it's not applicable to production (because there wouldn't be any point in "previewing" changes on the live website, because that's the same as just making the changes live).

File: charts/app-config/values-integration.yaml
Comment (sengi) [inline charts/app-config/values-integration.yaml:1708]: Sounds like perhaps this doesn't need to be a secret after all: https://github.com/alphagov/govuk-puppet/pull/8041

I couldn't find anything on the web about what knowing a `gtm_auth` value might allow someone to do, so I've asked UX measurement team.

Comment (sengi) [reply charts/app-config/values-integration.yaml:1708]: Verdict from UX measurement team: not secret 👍

Summary of my understanding of this now: if someone wanted to push fake data to our GA account (which they can do anyway), knowing this value would let them have that data be processed with the GA/GTM configuration that's being previewed on the integration/staging site as opposed to the configuration for the live site. In other words, it's useless for nefarious purposes.

**Comment by sengi** [comment]
RFAL

**Review by nsabri1 — APPROVED** [review]
LGTM - always feel weird when the label is relating to Auth that we have it as plain text. As you mentioned in standup you have discussed this with the relevant teams and they have given the green light.

**Pull request merged**
